ledger/lruaccts_test.go

```diff
@@ -195,3 +196,40 @@ func TestLRUAccountsOmittedPendingWrites(t *testing.T) {
 		require.Equal(t, persistedAccountData{}, acct)
 	}
 }
+
+func BenchmarkLRUAccountsWrite(b *testing.B) {
+	accounts := generatePersistedAccountData(0, 5000)
+	fillerAccounts := generatePersistedAccountData(5000, 100000)
+	baseAccts := make([]lruAccounts, b.N)
+	for i := 0; i < b.N; i++ {
+		baseAccts[i].init(logging.TestingLog(b), 10, 5)
+		for _, account := range fillerAccounts {
+			baseAccts[i].write(account)
+		}
+	}
+
+	b.ResetTimer()
+	for i := 0; i < b.N; i++ {
+		baseAcct := baseAccts[i]
+		for _, account := range accounts {
+			baseAcct.write(account)
+		}
+	}
+}
+
+func generatePersistedAccountData(startRound, endRound int) []persistedAccountData {
+	accounts := make([]persistedAccountData, endRound-startRound)
+	for i := startRound; i < endRound; i++ {
+
+		digest := crypto.Hash([]byte{byte(i)})
```

Comment:
This would create many entries.. but only 256 unique accounts.
I don't believe it was your intent, right ?

Reply:
Creating overlapping account between the two groups is fine, of course

Reply:
You are correct, thank you for noticing!

I've decided to avoid overlapping accounts, I hope this isn't an issue.